app/models/ability.rb

```diff
@@ -34,8 +34,8 @@ def not_signed_in
       conference.splashpage && conference.splashpage.public == true
     end
     # Can view the schedule
-    can [:schedule, :events], Conference do |conference|
-      conference.program.cfp && conference.program.schedule_public
+    can [:show, :schedule, :events], Conference do |conference|
```

Comment:
How is this related to the issue? 
:show gives you access to the show action of the Conference. How is this relevant with the schedule?